app/views/registration/new.html.erb

```diff
@@ -29,7 +29,7 @@
 
       <div class="uk-form-row">
         <%= f.label :room, class: 'uk-form-label form-label-required' %>
-        <%= f.number_field :room, class: 'uk-width-1-1 uk-form-large', placeholder: 'Szobaszám' %>
+        <%= f.text_field :room, class: 'uk-width-1-1 uk-form-large', placeholder: 'Szobaszám' %>
```

Comment:
The length could be validated through active record, because it is 10 char long in db and it is easier to generate application error (using text instead of number is more tempting to write longer texts). 

Although there is another card to extend the length and the validation could be added there too, so I can accept it.